src/cephadm/cephadm.py

```diff
@@ -7109,6 +7109,13 @@ def command_rm_daemon(ctx):
          verbosity=CallVerbosity.DEBUG)
     call(ctx, ['systemctl', 'disable', unit_name],
          verbosity=CallVerbosity.DEBUG)
+
+    # force remove rgw admin socket file if leftover
+    if daemon_type in ['rgw']:
+        rgw_asok_path = '/var/run/ceph/' + ctx.fsid + '/ceph-client.' + ctx.name + '.*.asok'
```

Comment:
LGTM

nit: plz consider using f-strings for the formatting instead

```suggestion
        rgw_asok_path = f'/var/run/ceph/{ctx.fsid}/ceph-client.{ctx.name}.*.asok'
```